vm/src/vm.rs

```diff
@@ -107,6 +108,18 @@ pub struct PySettings {
 
     /// sys.argv
     pub argv: Vec<String>,
+
+    /// Initialize VM after VM created
+    /// - Some(true):
+    ///     VM will be created and initialized with the importers, which require external
+    ///     filesystem access
+    /// - Some(false):
+    ///     VM will be created and initialized with the importers wihout external filesystem access
+    /// - None:
+    ///     VM will be created but not initialized, so it is ready to be injected with modules.
+    ///     After injection, the `initialize_as_external` or `initialize_without_external`
+    ///     should be called, such that the VM will be ready to use.
+    pub initialize_with_external_importer: Option<bool>,
```

Comment:
Maybe this could be its own enum to make the option more clear?
```rust
enum Initialization { NoInitialize, InitializeInternal, InitializeExternal }
```

Reply:
Thanks for your advice.  Use enum is much clear in this case.